node/services/config.go

```diff
@@ -59,3 +64,15 @@ func (cfg *Config) trustedHash() (tmbytes.HexBytes, error) {
 	}
 	return hex.DecodeString(cfg.TrustedHash)
 }
+
+// KeyConfig contains configuration details for construction of the keyring.Signer
+// that will be used to manage the node's account.
+type KeyConfig struct {
+	KeyringAccName string
```

Comment:
I guess this is the name for a default keyring account name. Do we really need a way to configure its name? It does not seem to change anything meaningful for users.

To handle a default Node key we can:
* Have a name constant "default" which is the default for everyone and this is the key that is autogenerated on the first start of the node, so users would need to top it up with tokens to use it. This is the most simplest approach
* Have an API that allows users to generate/import a key/account and set it as default with an API. This is an approach we should get to eventually. 

In both cases, I don't see why users would want to change their default keyring name through a config.

Reply:
Have taken the first approach. Resolved?